package.json

```diff
@@ -24,9 +28,23 @@
     "@testing-library/react": "^12.0.0",
     "babel-jest": "^27.0.6",
     "cypress": "^7.7.0",
+    "eslint": "^7.31.0",
+    "eslint-config-next": "^11.0.1",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-config-standard": "^16.0.3",
     "eslint-plugin-cypress": "^2.11.3",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.4.0",
+    "eslint-plugin-promise": "^5.1.0",
+    "eslint-plugin-react": "^7.24.0",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "eslint-plugin-security": "^1.4.0",
+    "identity-obj-proxy": "^3.0.0",
     "jest": "^27.0.6",
     "msw": "^0.32.0",
+    "prettier": "^2.3.2",
     "start-server-and-test": "^1.12.6"
   }
 }
```

Comment:
Linting, prettier, formatting script for initial commit and CI, some common eslint configs and plugins